### PR TITLE
39621 plugin backend design adjustment

### PR DESCRIPTION
**Pull request by dai-eastgate**
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/581
changed log : 
consistent width and design for inputs, dropdowns and info messages

### CODE REVIEW

File: css/admin.css [hunk 1]
```diff
@@ -496,4 +472,159 @@ body #post-body-content p.wp-clearfix:nth-child(1){
...
+	display: block !important;
+}
+
+.oo-poststuff .custom-input-field .onoffice-multiselect-popup label {
+	display: flex;
+	flex-direction: row-reverse;
+}
+
+.oo-poststuff .sortable-item .custom-language select,
+.oo-poststuff .sortable-item .custom-language label {
```
Comment (andernath) [inline css/admin.css:604]: I think this label is already styled in line 509. Or will it be overwrite by some other styling?

File: css/admin.css [hunk 2]
```diff
@@ -496,4 +472,159 @@ body #post-body-content p.wp-clearfix:nth-child(1){
...
+.oo-poststuff .sortable-item .custom-input-field:has(div.multiselect) {
+	display: block !important;
+}
+
+.oo-poststuff .custom-input-field .onoffice-multiselect-popup label {
+	display: flex;
+	flex-direction: row-reverse;
+}
+
+.oo-poststuff .sortable-item .custom-language select,
```
Comment (andernath) [inline css/admin.css:603]: Can .oo-poststuff .custom-language select move to the other block with 50%?
And is .custom-langiuge here really needed?

**Review by andernath — COMMENTED** [review]
@dai-eastgate please take a look inside the review

File: css/admin.css [hunk 3]
```diff
@@ -496,4 +472,159 @@ body #post-body-content p.wp-clearfix:nth-child(1){
...
+.oo-poststuff #viewrecordsfilter .custom-input-field,
+.oo-poststuff #viewrecordssorting .custom-input-field,
+.oo-poststuff #viewlayoutdesign .custom-input-field,
+.oo-poststuff #viewdocumenttypes p,
+.oo-poststuff #viewformspecific .custom-input-field,
+.oo-poststuff #geofields .custom-input-field,
+.oo-poststuff #sortableFieldsList .custom-input-field,
+.oo-poststuff #viewaccesscontrol .custom-input-field,
+.oo-poststuff #viewsimilarestates .custom-input-field,
+.oo-poststuff #onoffice-ajax .postbox .custom-input-field {
```
[the remark this answers is not in the document]
Comment (dai-eastgate) [reply css/admin.css:487]: > @dai-eastgate You are right the old html and css is a mess. :( And it is hard to improve it without wasting too much time.
> 
> Your changes in plugin-backend-design-adjustment-v2 look much better. Thank you for refactoring. Can you open a pull request for plugin-backend-design-adjustment-v2? I will test it :)

@andernath I will create a pull request for it when I complete the code and report back to you

**Pull request closed without merging**